package.json

```diff
@@ -17,6 +17,7 @@
     "eslint-config-airbnb": "^18.0.1",
     "eslint-config-prettier": "^6.5.0",
     "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-import-order": "^2.1.4",
     "eslint-plugin-jsx-a11y": "^6.2.3",
```

Comment:
could you add this as well?
https://www.npmjs.com/package/eslint-plugin-simple-import-sort

Reply:
I did